<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>498</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="labelLayout">
        <item>
         <widget class="QLabel" name="cacheLabel">
          <property name="text">
           <string>Cache folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="folderSizeLabel">
          <property name="text">
           <string>Cache folder size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="numberOfFilesLabel">
          <property name="text">
           <string>Number of files in memory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="portLabel">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="parametersLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="cacheFolderLineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectFolderButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearDirectoryButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cacheSizeSpinBox">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> GB</string>
          </property>
          <property name="value">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numberOfFilesSpinBox">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="portSpinBox">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="serversBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string>Remote Asset Cache Server</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="addNewServerButton">
         <property name="text">
          <string>Add server</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>364</width>
            <height>77</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="saveCancelLayout">
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
